app/models/rubygem.rb

```diff
@@ -275,7 +263,17 @@ def protected_days
   end
 
   def reverse_dependencies
-    self.class.reverse_dependencies(name)
+    self.class.joins("inner join versions as v on v.rubygem_id = rubygems.id
+      inner join dependencies as d on d.version_id = v.id").where("v.indexed = 't'
+      and v.position = 0 and d.rubygem_id = ?", id)
```

Comment:
So now we only revert the dependencies for latest version?
Should we change the method name? Or add docs?